packages/storybook/stories/Components/Pagination.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import { StoryObj } from "@storybook/react";
+import { PaginationItemsPerPage as ItemsPerPage, Pagination, PaginationProps } from "../";
+import { action } from "@storybook/addon-actions";
+
+const props: PaginationProps = {
```

Comment:
Without specifying the type here, `meta.args` cannot properly recognise the types of `itemsPerPage` and `itemsPerPageOptions`, but instead accepts any number.